src/lib/components/BottomSheet.svelte

```diff
@@ -1,4 +1,21 @@
-<div role="dialog" data-tid="bottom-sheet">
+<script lang="ts">
+  import { layoutBottomOffset } from "../stores/layout.store";
+  import { onDestroy } from "svelte";
+
+  onDestroy(() => ($layoutBottomOffset = 0));
+
+  // 1024 is the $breakpoint-large size. See also the CSS code of this component. On large screen the bottom sheet is not sticky.
+  const updateBottomOffset = () =>
+    ($layoutBottomOffset = innerWidth < 1024 ? height : 0);
```

Comment:
What do you think, does it make sense to expose breakpoints and read the value directly from the DOM (e.g. `getComputedStyle(document.documentElement).getPropertyValue("--breakpoint-large");`)?

Reply:
@mstrasinskis I would absolutely love that! Don't like to have that value hardcoded.

Can you take care of it in another PR?